ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/analyzers/DirectoryAnalyzerByTextTest.java

```diff
@@ -0,0 +1,93 @@
+package edu.usb.argos.ASTProcessor.analyzers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import edu.usb.argos.ASTProcessor.application.analyzers.IFileAnalyzer;
+import edu.usb.argos.ASTProcessor.application.exceptions.FileAnalyzerException;
+import edu.usb.argos.ASTProcessor.infrastructure.analyzers.directoryAnalyzers.DirectoryAnalyzerByText;
+
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
please avoid using this kind of imports, never import all the libraries using wildcards

Reply:
Done